reference/api/search.md

```diff
@@ -27,22 +27,22 @@ This is the preferred route to perform search when an API key is required, as it
 
 #### Body
 
-| Query Parameter                                       | Description                                        | Default Value |
-|-------------------------------------------------------|----------------------------------------------------|---------------|
-| **[q](#query-q)**                                     | Query string                                       | `""`          |
-| **[offset](#offset)**                                 | Number of documents to skip                        | `0`           |
-| **[limit](#limit)**                                   | Maximum number of documents returned               | `20`          |
-| **[filter](#filter)**                                 | Filter queries by an attribute's value             | `null`        |
-| **[facets](#facets)**                                 | Display the count of matches per facet             | `null`        |
-| **[attributesToRetrieve](#attributes-to-retrieve)**   | Attributes to display in the returned documents    | `["*"]`       |
-| **[attributesToCrop](#attributes-to-crop)**           | Attributes whose values have to be cropped         | `null`        |
-| **[cropLength](#crop-length)**                        | Maximum length of cropped value in words           | `10`          |
-| **[cropMarker](#crop-marker)**                        | String marking crop boundaries                     | `"…"`         |
-| **[attributesToHighlight](#attributes-to-highlight)** | Highlight matching terms contained in an attribute | `null`        |
-| **[highlightPreTag](#highlight-tags)**                | String inserted at the start of a highlighted term | `"<em>"`      |
-| **[highlightPostTag](#highlight-tags)**               | String inserted at the end of a highlighted term   | `"</em>"`     |
-| **[showMatchesPosition](#show-matches-position)**     | Return matching terms location                     | `false`       |
-| **[sort](#sort)**                                     | Sort search results by an attribute's value        | `null`        |
+| Query Parameter                                         | Description                                        | Default Value |
```

Comment:
Just wanted to point out that these are not "Query parameters", as they are not part of the query string. They are part of the JSON payload.

Some ideas for how we could rename this column: "Body parameter", "Body property", "Payload parameter", "Payload property"... or drop the prefix and simply name the column "Property" or "Parameter". WDYT @guimachiavelli @maryamsulemani97 ?

Reply:
I think just "parameter" works. We refer to these everywhere as "the `filter` search parameter" or "the `filter` parameter"

Reply:
Ackchyually…they are query parameters. You can make requests such as

`curl   -X GET 'http://localhost:7700/indexes/movies/search?q=batman&attributesToCrop=overview'`

And receive an adequate response. It's of course unpractical to say the least, especially when you have complex values like arrays and objects, but I think they do classify as query parameters.

Unless I'm missing something?

Reply:
Well, we are both right. The code sample you posted works, but only because it is using the GET endpoint.

With the POST endpoint, the so-called search parameters are part of the payload/body. However since GET does not accept a payload, they have to be query parameters when using GET /indexes/{index_uid}/search.

Reply:
My point was that since this table is describing the POST endpoint, we shouldn't call the column "Query parameters", we should call it "Body", "Parameter", "Field" or something similar.

Reply:
hmmm…although technically correct for `POST`, I'm not convinced differentiating them is helpful for users.

I'm fine with an unqualified "Parameter", though, which would translate to both POST and GET looking the same. Perhaps we could that for all query parameters tables across the reference?

Either way, this discussion is out of scope for this PR, right? I'm going to give my approval so we can merge this and we can continue talking in a dedicated PR.